docs/taskruns.md

```diff
@@ -678,12 +678,14 @@ spec:
   resources:
     inputs:
       - name: workspace
-        type: git
-  steps:
-    - name: config
-      image: ubuntu
-      command: ["/bin/bash"]
-      args: ["-c", "cat README.md"]
+        resourceSpec:
```

Comment:
This needs a little bit more to be a successful taskrun:

```yaml
        resourceSpec:
          type: git
          params:
            - name: url
              value: https://github.com/tektoncd/pipeline.git
```

And the `taskSpec` needs the following in addition to the steps:

```yaml
  taskSpec:
    resources:
      inputs:
      - name: workspace
        type: git
```